piicatcher/scanner.py

```diff
@@ -161,6 +171,9 @@ def data_scan(
         LOGGER.debug("Scanning column name %s", column.fqdn)
         if val is not None:
             for detector in detectors:
+                if column.pii_type is not None:
+                    continue
```

Comment:
Does this mean if the metadata scan has already determined the PII type of the column, then the data of that column will not be scanned any more?

Reply:
yup, but not sure if that should be the default or allow the datum scanning to override the piitype